src/pages/ReimbursementAccount/RequestorOnfidoStep.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import lodashGet from 'lodash/get';
+import {ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import * as BankAccounts from '../../libs/actions/BankAccounts';
+import Onfido from '../../components/Onfido';
+import compose from '../../libs/compose';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as ReimbursementAccountUtils from '../../libs/ReimbursementAccountUtils';
+import Growl from '../../libs/Growl';
+import reimbursementAccountPropTypes from './reimbursementAccountPropTypes';
+import CONST from '../../CONST';
+
+const propTypes = {
+    /** Bank account currently in setup */
+    /* eslint-disable-next-line react/no-unused-prop-types */
+    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
+    onfidoToken: PropTypes.string,
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    onfidoToken: '',
+};
+
+class RequestorOnfidoStep extends React.Component {
+    constructor(props) {
+        super(props);
+        this.submit = this.submit.bind(this);
+        this.state = {
+            onfidoData: lodashGet(props, ['reimbursementAccount', 'achData', 'onfidoData'], ''),
```

Comment:
Do we actually need to put onfidoData in the state? I feel like we could use no state, and then just call submit with onfidoData on onfido success.